exercises/atbash-cipher/src/test/java/AtbashTest.java

```diff
@@ -1,73 +1,59 @@
+import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.experimental.runners.Enclosed;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameters;
-
-import java.util.Arrays;
-import java.util.Collection;
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(Enclosed.class)
 public class AtbashTest {
 
-    @RunWith(Parameterized.class)
-    public static class EncodeTest {
-        private String plaintext;
-        private String ciphertext;
-
-        @Parameters(name = "{index}: expected plaintext \"{0}\" to encode to ciphertext \"{1}\".")
-        public static Collection<Object[]> data() {
-            return Arrays.asList(new Object[][]{
-                    {"yes", "bvh"},
-                    {"no", "ml"},
-                    {"OMG", "lnt"},
-                    {"O M G", "lnt"},
-                    {"mindblowingly", "nrmwy oldrm tob"},
-                    {"Testing,1 2 3, testing.", "gvhgr mt123 gvhgr mt"},
-                    {"Truth is fiction.", "gifgs rhurx grlm"},
-                    {"The quick brown fox jumps over the lazy dog.", "gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt"}
-            });
-        }
+    private Atbash atbash;
 
-        public EncodeTest(String plaintext, String ciphertext) {
-            this.plaintext = plaintext;
-            this.ciphertext = ciphertext;
-        }
+    @Before
+    public void setup() {
+        atbash = new Atbash();
+    }
 
+    @Test
+    public void testEncodingSingleWord() {
+        assertEquals("bvh", atbash.encode("yes"));
+        assertEquals("ml", atbash.encode("no"));
```

Comment:
As discussed in PR #1305, I would prefer it if these were split into one assertion per test to match the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/atbash-cipher/canonical-data.json) 🙂

Reply:
Ok sure, I will make the changes accordingly.